<?xml version="1.0" ?>
<launch>
    <rosparam file = "$(find pid_control)/config/system_params.yaml" command = "load"  />
    <rosparam file = "$(find pid_control)/config/setpoint_params.yaml" command = "load"  />
    <rosparam file = "$(find pid_control)/config/control_params.yaml" command = "load"  />

    <node name="Set_Point_Generator" pkg="pid_control" type="set_point_generator.py" output="screen"/>
    <node name="Controller" pkg="pid_control" type="controller.py" output="screen"/>
    <node name="Motor_Sim" pkg="pid_control" type="system.py" output="screen"/>
    <node name="rqt_plot" pkg="rqt_plot" type="rqt_plot" output="screen"  args="/motor_output/output /set_point/entrada "/>
    <node name="rqt_graph" pkg="rqt_graph" type="rqt_graph" output="screen"/>
</launch>
